BUILD.gn

```diff
@@ -1194,10 +1196,27 @@ template("dist_zip") {
     write_runtime_deps = _runtime_deps_file
   }
 
+  action(_strip_binaries_target) {
+    forward_variables_from(invoker, [ "testonly" ])
+    script = "//electron/script/strip-binaries.py"
+    deps = [
+      ":$_runtime_deps_target",
+    ]
+    args = [
+      "--directory",
+      rebase_path(root_out_dir),
+      "--target-cpu",
+      target_cpu,
+    ]
+    outputs = [
+      "$target_gen_dir/$_strip_binaries_target/.strip",
```

Comment:
GN hates steps that modify files in the output, and this will probably lead to weird situations where GN doesn't properly detect that a rebuild is required. Can we do this another way?

Reply:
Not without changing how literally everything else works.

I set it up like this so it "outputs" a file that it never actually creates.  This way GN will always run the script (never think it's done)

Reply:
Not the best, but it works for now and doesn't require substantial changes to everything else